main/guides/getting-started/syncing-up.md

```diff
@@ -1,99 +1,13 @@
 # Syncing Up on Mainnet
 
-To run a node on the `agoric-3` network from scratch, please follow the
-steps below.
+Tune in to [Network Upgrades Governance](https://community.agoric.com/c/governance/network-upgrades/15) in the [Agoric Community Forum](https://community.agoric.com/) for upcoming changes to mainnet node operations, including deployment of
```

Comment:
past and upcoming changes?

Reply:
going with just "changes", no "upcoming" qualifier